tests/template_tests/test_context.py

```diff
@@ -96,6 +110,17 @@ def test_flatten_context(self):
             'a': 2, 'b': 4, 'c': 8
         })
 
+    def test_flatten_context_with_context(self):
+        """
+        #24765 -- Verifies Using Context with Flatten Works
```

Comment:
suggested wording: `Context.push() with a Context argument should work (#24765).`